pkg/endpoint/log.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2017 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package endpoint
+
+import (
+	"github.com/cilium/cilium/pkg/logfields"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func (e *Endpoint) log() *log.Entry {
+	return log.WithFields(log.Fields{
+		logfields.EndpointID:  e.ID,
+		logfields.ContainerID: e.DockerID,
+		// REVIEW Is this a valid association?
+		"iteration": e.getIteration(),
+	})
+}
+
+func (e *Endpoint) getIteration() uint64 {
```

Comment:
@tgraf should be `e.Consumable.Iteration` or `e.policyRevision`?

Reply:
The iteration is unused now, it predated policy revision. We can remove the iteration field and use the policy revision in the logs.